lib/lndclient/LndClient.ts

```diff
@@ -26,14 +26,17 @@ interface GrpcResponse {
 class LndClient extends BaseClient {
   private lightning!: LightningClient;
   private meta!: grpc.Metadata;
+  private config!: LndClientConfig;
```

Comment:
Rather than keep the entire config object as a property on the class, I think I'd prefer just keeping `host` and `port`. Even better maybe just have a property called `uri` which is the `${host}:${port}` concatenation and set it in the constructor.